client/less/main.less

```diff
@@ -1155,6 +1155,11 @@ code {
   }
 }
 
+// make btn-default hover more visible
+.btn-default:hover {
+  background-color: @brand-primary !important;
```

Comment:
~~`.btn-default:hover`~~ `.btn-default:hover, .btn-default:active`

You don't need `!important` here.